src/main/java/be/seeseemelk/mockbukkit/entity/LivingEntityMock.java

```diff
@@ -629,6 +629,5 @@ public void setInvisible(boolean invisible)
 	public boolean isInvisible()
 	{
 		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
-	}
+		throw new UnimplementedOperationException()
```

Comment:
The build is failing because of this change. Is this intended?

Reply:
Missing semicolon XD
Thanks for noticing @poma123, I simply resolved the conflicts on mobile and didn't even bother to check the CI results.